src/cli/cli.cc

```diff
@@ -96,8 +106,20 @@ StatusCode Install(AkliteClient &client, int version, const std::string &target_
     }
   }
   if (target.IsUnknown()) {
-    LOG_ERROR << "No Target found; version: " << (version == -1 ? "latest" : std::to_string(version))
-              << ", hardware ID: " << client.GetConfig().get("provision.primary_ecu_hardware_id", "")
+    std::string target_string;
+    if (version == -1 && target_name.empty()) {
+      target_string = " version: latest,";
+    } else {
+      if (!target_name.empty()) {
+        target_string = " name: " + target_name + ",";
+      }
+      if (version != -1) {
```

Comment:
This `if` is redundant. Maybe the ternary operator `<>?<>:<>` will work a bit better?

Reply:
please, ignore, I misunderstood the code in the first place.